examples/problems/1300_sequential_alignment.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p> </p><p>Before this, talk about the policies, i.e. say which options there are (StarPolicy, SeqeuentialPolicy), similarly to what we had in the issue. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>